packages/server/lib/browsers/cdp_automation.ts

```diff
@@ -174,18 +174,29 @@ const ffToStandardResourceTypeMap: { [ff: string]: ResourceType } = {
   'webmanifest': 'manifest',
 }
 
+interface CdpAutomationOptions {
+  onBrowserPreRequest?: (BrowserPreRequest) => void
+}
+
 export class CdpAutomation {
-  constructor (private sendDebuggerCommandFn: SendDebuggerCommand, onFn: OnFn, private automation: Automation) {
+  frameTree?: Protocol.Page.FrameTree
+
```

Comment:
doesn't appear to be used

```suggestion
  frameTree?: Protocol.Page.FrameTree

```